src/improvements/tasks/MultisigTask.sol

```diff
@@ -37,67 +37,24 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
     /// This state variable is always set in the `_taskSetup` function
     address public parentMultisig;
 
-    /// @notice struct to store the addresses that are expected to have storage accesses
-    EnumerableSet.AddressSet internal _allowedStorageAccesses;
-
-    /// @notice struct to store the addresses that are expected to have balance changes
-    EnumerableSet.AddressSet internal _allowedBalanceChanges;
-
-    /// @notice Struct to store information about a token/Eth transfer
-    /// @param to The address of the recipient
-    /// @param value The amount of tokens/Eth to transfer
-    /// @param tokenAddress The address of the token contract
-    struct TransferInfo {
-        address to;
-        uint256 value;
-        address tokenAddress;
-    }
-
-    /// @notice Struct to store information about a state change
-    /// @param slot The storage slot that is being updated
-    /// @param oldValue The old value of the storage slot
-    /// @param newValue The new value of the storage slot
-    struct StateInfo {
-        bytes32 slot;
-        bytes32 oldValue;
-        bytes32 newValue;
-    }
-
-    /// @notice Enum to determine the type of task
-    enum TaskType {
-        L2TaskBase,
-        SimpleTaskBase,
-        OPCMTaskBase
-    }
-
-    /// @notice state changes during task execution
-    mapping(address => StateInfo[]) internal _stateInfos;
-
-    /// @notice addresses whose state is updated in task execution
-    EnumerableSet.AddressSet internal _taskStateChangeAddresses;
-
-    /// @notice stores the gnosis safe accesses for the task
-    VmSafe.StorageAccess[] internal _accountAccesses;
-
     /// @notice starting snapshot of the contract state before the calls are made
-    uint256 internal _startSnapshot;
-
-    /// @notice Task TOML config file values
-    struct TaskConfig {
-        string[] allowedStorageKeys;
-        string[] allowedBalanceChanges;
-        string safeAddressString;
-    }
+    uint256 public startSnapshot;
 
     /// @notice configuration set at initialization
     TaskConfig public config;
 
     /// @notice flag to determine if the task is being simulated
-    uint256 private _buildStarted;
+    uint256 public buildStarted;
```

Comment:
Why do we make this public? I assumed it was private to avoid accidentally mutating this variable in child contracts

Reply:
For the forge StdStorage functionality I added. Can it work on private state variables? I didn't think it could.

Reply:
It will if you have a getter for that var, just tested with this:

```solidity
contract Q is Test {
  using stdStorage for StdStorage;

    function test_StorageAccess() external {
        R r = new R();

        stdstore
            .target(address(r))
            .sig(r.getX.selector)
            .checked_write(42);

        assertEq(r.getX(), 42);
    }
}

contract R  {
  uint private x;

  function getX() public view returns (uint) {
    return x;
  }

  function set(uint _x) public {
    x = _x;
  }
}
```

Reply:
awesome, I've made this change: https://github.com/ethereum-optimism/superchain-ops/pull/1059/commits/5897a386b6dc3495f2de40cf6166a7168a550f46